src/Panel.vue

```diff
@@ -124,30 +120,50 @@
       },
       bottomSwitch: {
         type: Boolean,
-        coerce: coerce.boolean,
         default: true
       }
     },
-    data () {
-      return {
-        onHeaderHover: false
-      }
-    },
     computed: {
+      // Vue 2.0 coerce migration
+      expandableBool: function () {
```

Comment:
Doing `expandableBool() {` is sufficient, same for the rest of the migrated coerce methods.